examples/widget_sparql.html

```diff
@@ -19,11 +19,6 @@
      * The class that adds the display of a drop-down menu containing clustering functionalities when a node is clicked
      */
     class ContextMenuGraphClustering {
-      /**
-       * Create a new Context Menu
-       * 
```

Comment:
Why did you delete this part of the commentary?

Reply:
Typing in HTML is not a relevant thing: it produces an error